src/components/custom/_dnumbers.tsx

```diff
@@ -50,18 +51,18 @@ const Title = styled(Header)`
 const DNumbers = ({ items, justify }: DNumbersProps) => {
     return (
         <StyledContainer justify={justify || 'space-between'}>
-            {items.map((item) => (
-                <NumberWrapper
-                    key={
-                        typeof item.subtitle === 'string'
-                            ? item.subtitle
-                            : item.subtitle.props.translate_text
-                    }
-                >
+            {items.map(({ subtitle, title }) => (
+                <NumberWrapper key={title}>
                     <Title as="p" type="page-title" align="center">
-                        {item.title}
+                        {title.includes('_t_') ? (
+                            <Localize translate_text={title as TString} />
+                        ) : (
+                            title
+                        )}
                     </Title>
-                    <StyledText align="center">{item.subtitle}</StyledText>
+                    <StyledText align="center">
+                        <Localize translate_text={subtitle} />
```

Comment:
Hi @niloofar-deriv I have one question. 
As you are defining type in line number 58 then why not here as well 😬 
Like `subtitle as string` 👀

Reply:
We shouldn't use type casting in our projects and the default type of `subtitle` is `TString`
The reason I used type casting in line 58 was because of the type of `title`, it can be both `string` and `TString` so I had to use type casting to fix localize ts error, and with the include `_t_` condition I'm sure that the type is that!